ArcTouch.Droid/Views/HomeActivity.cs

```diff
@@ -12,12 +12,12 @@
 using Android.Support.V7.App;
 using Android.Views;
 using Android.Support.V7.Widget;
-using GenesisAuto.Core.ViewModels;
+using ArcTouch.Core.ViewModels;
 using MvvmCross.Platforms.Android.Binding.Views;
 using MvvmCross.Platforms.Android.Views;
 using Android.Support.V4.View;
 
-namespace GenesisAuto.Droid.Views
+namespace ArcTouch.Droid.Views
 {
     [Activity(Label = "GitHub Awesome", Theme = "@style/AppTheme") ]
     public class HomeActivity : BaseActivity<HomeViewModel>
```

Comment:
Following the MVVM specification, the view should be unaware of the actual ViewModel implementation.